### ISSUE TITLE
How to change the orientation?

### ISSUE THREAD

**Issue by karamelflan** [jlam55555/veikk-linux-driver#52]
I didn't understand the comment about changing the orientation and I am using a vertical monitor (90 degrees clockwise from default landscape orientation). what should I change?

**Comment by karamelflan** [comment]
nevermind I didn't read

**Comment by karamelflan** [comment]
Even so, I made a bit of workaround to make the portrait mode to still work even after reboot.
Changing the /sys/module/veikk/orientation only creates a temporary buffer which, even if you change it from 0 to 3, after a reboot, it changes back to 0. I'm no tech_savvy, not even a beginner in programming, but I figured out that in the veikk.h file; if you change the order of the:

enum veikk_orientation {
    VEIKK_OR_DFL = 0,
    VEIKK_OR_CCW,
    VEIKK_OR_FLIP,
    VEIKK_OR_CW
};

to:

enum veikk_orientation {
    VEIKK_OR_CW = 0,
    VEIKK_OR_CCW,
    VEIKK_OR_FLIP,
    VEIKK_OR_DFL
};

and reinstall it, it then just works flawlessly.

This is only a workaround I did because the GUI doesn't *for me* because I'm no expert, and even the Qt thingy needed for the GUI was giving too much error.
Only for those linux users who have the same problem as me.

Linux Mint Mate 20
Linux <username>-Aspire-ES1-572 5.4.0-48-generic #52-Ubuntu SMP Thu Sep 10 10:58:49 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
